js/jquery.terminal.js

```diff
@@ -1357,6 +1358,8 @@
                     } else {
                         if (command !== '' && position > 0) {
                             self['delete'](-1);
+                            var str = $('.clipboard').val()
+                            $('.clipboard').val(str.substring(0, str.length - 1));
```

Comment:
Yeah, it should be more graceful,when I was stucked I just debug into it and find the dirty but easy way out, did not pay attention to many other code around it. I d like to help if you want.